testcafe/tests/helpers/data.js

```diff
@@ -0,0 +1,5 @@
+export default class Data {
+  constructor() {
+    this.sharingLink = ''
```

Comment:
Are we planning on adding things to this class? Because as it stands it's not very useful 😛

Reply:
It will be usefull for more sharing scenario when I need to get some info from one user and pass it to another user.

Reply:
Mmh ok. It's not blocking for me, but I think it's better not to anticipate these things too much :)